doc/source/examples/analysis/hydrogen_bonds/hbonds-lifetimes.ipynb

```diff
@@ -100,7 +100,7 @@
     {
```

Comment:
<p>Why the change here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MDAnalysis/UserGuide/pull/263/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>